benchmarks/benchmarks/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "thread-pools-benchs",
+  "version": "1.0.0",
+  "description": "This project contains benchmarks for the most used and popular Node.js thread/cluster pools implementations",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "pioardi",
+  "license": "MIT",
+  "dependencies": {
+    "benchmark": "^2.1.4",
+    "poolifier": "^2.0.0-beta.5",
```

Comment:
Bump to the latest beta, and it will work better.

Reply:
I will do directly later, to test my performance enhancements I am now referring to a local path.
But thank you to catch anyway